components/DropDownMenu.js

```diff
@@ -137,13 +144,14 @@ class DropDownMenu extends Component {
 
   renderSelectedOption() {
     const { style, titleProperty } = this.props;
+    const getIcon = (icon = drop-down) => (_.isString(icon) ? <Icon name={icon} /> : <icon />)
```

Comment:
Define this function outside of a render function. This will unnecessarily create a new function on each render.